tenseal/sealapi/sealapi_encrypt.cpp

```diff
@@ -204,9 +204,7 @@ void bind_encrypt_decrypt(pybind11::module &m) {
         .def("parms_id", py::overload_cast<>(&Plaintext::parms_id, py::const_))
         .def("scale", py::overload_cast<>(&Plaintext::scale, py::const_))
         .def("set_scale",
-             [](Plaintext &obj, double scale) {
-                 obj.scale() = scale;
-             })
+             [](Plaintext &obj, double scale) { obj.scale() = scale; })
```

Comment:
Can't we have something like property `.def_property("name", &Pet::getName, &Pet::setName)` from https://pybind11.readthedocs.io/en/stable/classes.html#instance-and-static-fields ?

Reply:
Nice, I didn't know about that